constraintsolver/include/GSolver.h

```diff
@@ -99,7 +99,7 @@ class GSolver {
     double _utilitySignificanceThreshold = 1E-22;
     // Random rand;
     int _dim;
-    shared_ptr<vector<shared_ptr<vector<double>>>> _limits;
```

Comment:
What was the motivation behind moving limits from a member to a parameter in the member functions?

Reply:
There is no need for it be a member. I simply wanted to avoid both shared_ptr and copy into member.
In the future, these things should move into a struct that ties together all things needed for solving and is simply passed by reference between different functions.